shared/chat/conversation/messages/react-button/emoji-picker/index.tsx

```diff
@@ -7,11 +7,22 @@ import {chunk} from 'lodash-es'
 import {memoize} from '../../../../../util/memoize'
 
 // defer loading this until we need to, very expensive
-const getData = memoize(() => {
+const getData = memoize((topReacjis: Array<string>) => {
```

Comment:
i don't think you want to memoize these two things together. keep them as 2 different things